modules/extensions-spring-support/src/test/resources/models/subtypes.json

```diff
@@ -49,7 +49,25 @@
     {
       "baseType": {
         "format": "java",
-        "type": "@ref:org.mule.test.heisenberg.extension.model.Weapon"
+        "type": "Object",
+        "annotations": {
+          "typeId": "org.mule.test.heisenberg.extension.model.Weapon",
```

Comment:
this should still be a ref. Otherwise there could be an inconsistency between what's described here and what's on the original extension